README.md

```diff
@@ -1 +1,36 @@
+<div style="text-align:center"><img src="docs/images/YAML_Runner_Logo_250.png"/></div>
+
 # Yaml Runner
+
+Yaml Runner is a python library that provides a command-line interface to run commands stored in a yaml file.
+
+
+## Installatation
+
+```
+pip install git+https://github.com/rdkcentral/yaml_runner.git@master
+```
+
+## Usage
+
+The simplest usages for this library is to use it to run it in a main function. See [yaml_runner_run.py](examples/yaml_runner_run.py)
+
+However, for more advanced usage this library can be extended. The simplest way to do this is to inherit it in a class and extend it there.
```

Comment:
This library can also be extended for more advanced usages. The easiest way is to inherit it in a class and extend it there.